src/Dapper.AOT.Analyzers/CodeAnalysis/DapperAnalyzer.cs

```diff
@@ -155,6 +160,39 @@ public void OnOperation(OperationAnalysisContext ctx)
                                 ValidatePropertyUsage(ctx, assignment.Value, false);
                             }
                         }
+                        break;
+                    case OperationKind.VariableDeclaration when ctx.Operation is IVariableDeclarationOperation variableDeclarationOperation
+                        && variableDeclarationOperation.Declarators.FirstOrDefault() is { } declarator
+                        && declarator.Initializer?.Value is IObjectCreationOperation objectCreationOperation:
+
+                        var ctor = objectCreationOperation.Constructor;
+                        var receiverType = ctor?.ReceiverType;
+
+                        if (ctor is not null && receiverType is
+                            {
+                                Name: "SqlCommand",
+                                ContainingNamespace:
+                                {
+                                    Name: "SqlClient",
+                                    ContainingNamespace:
+                                    {
+                                        Name: "Data",
+                                        ContainingNamespace:
+                                        {
+                                            Name: "Microsoft",
+                                            ContainingNamespace.IsGlobalNamespace: true
```

Comment:
Assuming we limit to SqlClient: there's 2 of them - System.Data and Microsoft.Data

Reply:
supported both System and Microsoft SqlClients